wfl/DBManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Threading.Tasks;
+using MongoDB.Driver;
+using wfl.Models;
+
+namespace wfl
```

Comment:
Normally namespaces start with a capital letter and begin with the company so `Symphono.Wfl` in this case.